editor/src/core/model/performance-scripts.tsx

```diff
@@ -331,6 +339,197 @@ export function useTriggerSelectionPerformanceTest(): () => void {
   return trigger
 }
 
+export function useTriggerAbsoluteMovePerformanceTest(): () => void {
+  const dispatch = useEditorState(
+    React.useCallback((store) => store.dispatch as DebugDispatch, []),
+    'useTriggerAbsoluteMovePerformanceTest dispatch',
+  )
+  const allPaths = useRefEditorState(
+    React.useCallback((store) => store.derived.navigatorTargets, []),
+  )
+  const metadata = useRefEditorState(React.useCallback((store) => store.editor.jsxMetadata, []))
+  const selectedViews = useRefEditorState(
+    React.useCallback((store) => store.editor.selectedViews, []),
+  )
+  const trigger = React.useCallback(async () => {
+    const initialTargetPath = [...allPaths.current].sort(
+      (a, b) => EP.toString(b).length - EP.toString(a).length,
+    )[0]
+    // This is very particularly tied to the test project, we _really_ need to pick the
+    // right element because our changes can cause other elements to end up on top of the
+    // target we want.
```

Comment:
Is this comment still accurate (specifically the part about it being tied to the test project itself) now that the test is no longer picking a very specific element?

Reply:
Yes, the steps of taking the longest path, going up two levels and then picking the last (in order) grandchild of that grandparent are just a programmatic way of getting the same static path that was originally in here.